tests/geometry/test_constrain_geometry.py

```diff
@@ -1,14 +1,107 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-"""
-Created on Wed Nov 29 09:13:33 2017
-
-@author: Eirik Keilegavlens
-"""
+"""This file contains testing of funcionality found within
+porepy.geometry.constrain_geometry.py"""
 
 import numpy as np
 
 from porepy import constrain_geometry
+from tests.test_utils import compare_arrays
+
+
+class TestBasic:
+    """
+    Various tests of intersect_polygon_lines.
+
+    """
+
+    def test_convex_polygon(self):
```

Comment:
There's not a lot of documentation here. I think that's fine, but let's at least improve test function names. In this case something like test_constrain_lines_by_polygon.